base/commands/migration/migration_stages.go

```diff
@@ -76,6 +77,15 @@ func createMigrationStages(ctx context.Context, ec plug.ExecContext, ci *hazelca
 					case StatusCanceled, StatusCanceling:
 						execErr = clcerrors.ErrUserCancelled
 						break statusReaderLoop
+					case StatusInProgress:
+						rt, cp, err := fetchOverallProgress(ctx, ci, migrationID)
+						if err != nil {
+							ec.Logger().Error(err)
+							status.SetText("Unable to calculate remaining duration and progress")
+						} else {
+							status.SetProgress(cp)
+							status.SetRemainingDuration(rt)
```

Comment:
What happens if the progress status is received after setting the ""Unable to calculate remaining duration and progress" text? Is it changed to "XXX is in progress?

Reply:
It is a bit hard to run this scenario on my local environment. I was expecting it to be updated with the stage's ProgressMsg. Is it wrong?

Reply:
No, progress message is set once, before the stage starts executing, see: clc/ux/stage/stage.go:152

Reply:
ok, this commit fixes it: https://github.com/hazelcast/hazelcast-commandline-client/pull/423/commits/465a2e06ed87b8183027d0514261a31517b2dd2e